packages/babel-preset-typescript/src/index.js

```diff
@@ -15,6 +15,10 @@ export default declare(
   ) => {
     api.assertVersion(7);
 
+    // Technically we could use the "semver" package here, but (for exmaple)
+    // parseFloat("4.23.6") returns 4.23 so it's "good enough"
+    const BABEL_SUPPORTS_EXTENSIONS_OPTION = parseFloat(api.version) >= 7.11;
```

Comment:
# :warning: 

- Before merging this PR, I need to add a warning in `new-version` in `Makefile` to replace 7.11 with the actual new version.